containers/Dockerfile.centos7-foss2020a

```diff
@@ -0,0 +1,12 @@
+FROM easybuilders/test:centos7-eb421-gcc930
+USER root
+RUN yum install -y rdma-core-devel openssl
+USER easybuild
+# install software & accompanying modules
+RUN (eb foss-2020a.eb --robot || tail -100 `eb --last-log`) \
+  # clean up everything in /scratch
+  && rm -rf /scratch/*
+# finalize /easybuild, no changes can be made in /easybuild that persist!
+VOLUME /easybuild
```

Comment:
If you want to be a good guy, you might want to use that to inform people:
```
ONBUILD RUN echo ">>> /data is defined as a VOLUME and thus won't persist! in subsequent builds"
```

Like this:
```
$ cat Dockerfile 
FROM busybox AS build
ONBUILD RUN echo ">>> /data is defined as a VOLUME and thus won't persist! in subsequent builds"

FROM build
RUN echo 1
$ docker build -t test .
Sending build context to Docker daemon  8.192kB
Step 1/4 : FROM busybox AS build
latest: Pulling from library/busybox
d9cbbca60e5f: Pull complete 
Digest: sha256:836945da1f3afe2cfff376d379852bbb82e0237cb2925d53a13f53d6e8a8c48c
Status: Downloaded newer image for busybox:latest
 ---> 78096d0a5478
Step 2/4 : ONBUILD RUN echo ">>> /data is defined as a VOLUME and thus won't persist! in subsequent builds"
 ---> Running in b093fa75c12e
Removing intermediate container b093fa75c12e
 ---> 6bc38005ea75
Step 3/4 : FROM build
# Executing 1 build trigger
 ---> Running in c5f4ee8c763a
>>> /data is defined as a VOLUME and thus won't persist! in subsequent builds
Removing intermediate container c5f4ee8c763a
 ---> 00c2421400ff
Step 4/4 : RUN echo 1
 ---> Running in 041a1e9986f7
1
Removing intermediate container 041a1e9986f7
 ---> 34c243ce0522
Successfully built 34c243ce0522
Successfully tagged test:latest
```

Reply:
even though I would not consider that best practice or something of that sort.

Reply:
And to prevent people from using the image in subsequent builds:
```
ONBUILD RUN exit 1
```